<ui version="4.0" >
 <class>programShortcutEditor</class>
 <widget class="QDialog" name="programShortcutEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Shortcut Editor</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QPushButton" name="iconButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="iconSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Name:</string>
     </property>
     <property name="buddy" >
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="nameEdit" />
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Executable:</string>
     </property>
     <property name="buddy" >
      <cstring>executableEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLineEdit" name="executableEdit" />
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="executableBrowseButton" >
     <property name="toolTip" >
      <string>Browse</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../icons/icons.qrc" >:/icons/browse.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Arguments:</string>
     </property>
     <property name="buddy" >
      <cstring>argsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QLineEdit" name="argsEdit" />
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Wine Dll Overrides ($WINEDLLOVERRIDES):</string>
     </property>
     <property name="buddy" >
      <cstring>wineDllOverridesEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QLineEdit" name="wineDllOverridesEdit" />
   </item>
   <item row="8" column="0" colspan="3" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Wine debug options ($WINEDEBUG):</string>
     </property>
     <property name="buddy" >
      <cstring>wineDebugEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3" >
    <widget class="QLineEdit" name="wineDebugEdit" />
   </item>
   <item row="10" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="terminalBox" >
        <property name="text" >
         <string>Show debugging output in a terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dosBox" >
        <property name="text" >
         <string>This is a Windows CUI (DOS) application</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>programShortcutEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>programShortcutEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
